CONTRIBUTORS.rst

```diff
@@ -160,6 +160,7 @@ Listed in alphabetical order.
   Meghan Heintz            `@dot2dotseurat`_
   Mesut Yılmaz             `@myilmaz`_
   Michael Gecht            `@mimischi`_                  @_mischi
+  Michael Karamuth         `@joshtensibly`_               
```

Comment:
This is creating a broken link, we need to add the target for `@joshtensibly` firther down in the file.